### DISCUSSION TITLE
Arduino CLI couldn't find the library

### DISCUSSION THREAD

**Discussion by DRSDavidSoft** [mathieucarbou/ESPAsyncWebServer#89] [General]
Hi there,

This might be a trivial question, but I'm unable to install the library using `arduino-cli` and the [`sketch.yaml`](https://github.com/arduino/arduino-cli/blob/master/docs/sketch-project-file.md) file.

I have tried the following:

- **`mathieucarbou/ESPAsyncWebServer (3.2.2)`**
- **`mathieucarbou/ESPAsyncWebServer @ 3.2.2`**

Which gives me: 
```
Can't open sketch: error loading sketch project file: invalid library directive: mathieucarbou/ESPAsyncWebServer (3.2.2)
```

And **`ESPAsyncWebServer (3.2.2)`** which gives me:

```
Downloading library ESPAsyncWebServer@3.2.2
Library ESPAsyncWebServer@3.2.2 not found
Error initializing instance: Library 'ESPAsyncWebServer' not found
```

Before this, I used to either uncompress the zip file or directly clone the git repo to the libraries directory.

I was wondering if you might have some clues regarding the cli, as the IDE I believe allows users to search for and install the fork.

Thanks!

**Comment by mathieucarbou** [comment]
Yes this is normal. Arduino are syncing the tags each night. PlatformIO is better at managing dependencies and Vscode is a better IDE.
For now, as a workaround, you can point to the GitHub directly and use: unsafe install like this example:

https://github.com/mathieucarbou/ESPAsyncWebServer/blob/main/.github/workflows/ci.yml#L58

**Reply by DRSDavidSoft** [reply]
Thank you, also I guess with the deprecation of the Arduino for VS Code extension (https://github.com/microsoft/vscode-arduino/issues/1757) I have to finally migrate to PlatformIO, I see the package and dependency management as a whole is quite better as well!

**Reply by DRSDavidSoft** [reply]
Excellent CI workflow, it's quite useful to incorporate it into my build system. Thanks for the info! 👍🏻
